internal/querynode/result.go

```diff
@@ -35,6 +35,11 @@ import (
 	"github.com/milvus-io/milvus/internal/util/typeutil"
 )
 
+const (
+	unlimited int    = -1
+	maxStr    string = "&*"
```

Comment:
Is there a reference for this value?

Reply:
No reference, just a special value to indicate it's the max string.